scripts/checkModels.py

```diff
@@ -285,7 +285,7 @@ def initialize_log(verbose, log_file_name='checkModels.log'):
     logging.basicConfig(
         filename=log_file_name,
         filemode='w',
-        format='%(name)s=%(message)s',
```

Comment:
What does this change mean? What does it look like before?

Reply:
It used to print something like `checkModels=` at the start of every line of the log file. Not helpful and just made it harder to read.